modules/workflow-service-remote/src/main/java/org/opencastproject/workflow/remote/WorkflowServiceRemoteImpl.java

```diff
@@ -722,4 +725,20 @@ public void cleanupWorkflowInstances(int lifetime, WorkflowState state) throws W
     }
     throw new WorkflowDatabaseException("Unable to successfully request the workflow cleanup endpoint");
   }
+
+  @Override
+  public Map<String, Map<String, String>> getWorkflowStateMappings() {
+    HttpGet get = new HttpGet("/statemappings.json");
+    HttpResponse response = getResponse(get);
+    try {
+      if (response != null) {
+        return (Map<String, Map<String, String>>) new JSONParser().parse(IOUtils.toString(response.getEntity().getContent(), "utf-8"));
```

Comment:
I am using JSON here while the rest of this class seems to use xml. However, the counterpart (WorkflowRestService) seems to provide JSON *and* xml for most endpoints. I opted in for JSON for the getWorkflowStateMappings() implementation since that was very easy to implement. However, I am not really happy with this. 

@staubesv @KatrinIhler any thoughts?

Reply:
@krinnewitz I don't know why those endpoints deliver both JSON and XML... likely, people missed the point that this would double the testing effort in case somebody would test this ;-)

I don't see an advantage in adding XML. In case somebody would actually want to use it, it could be added anytime.

Reply:
Same.